Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,14 +1,40 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.IO;
-using Console = System.Console;
 
 namespace BootCamp.Chapter
 {
     class Program
     {
-        static void Main(string[] args)
+        public static void Main(string[] args)
```

Comment:
The Main should only call the Lesson3 class. Create a new file Lesson3.cs for that class :)

Reply:
Can you clarify what you mean by this? I am working on the HW 3 branch. This shouldn't be main. Then what should I put in the Lesson3.cs file? My main class? Wouldn't that be most of my file?

Reply:
Alright I think I understand. I have put everything into the new Lesson3.cs file.